File: KuantumPYTHON/karanlik_madde.py
from kuantum_nesnesi import KuantumNesnesi
from kritik_arayuzu import IKritik

class KaranlikMadde(KuantumNesnesi, IKritik):

    def __init__(self, ID, stabilite, tehlike_seviyesi):
        super().__init__(ID, stabilite, tehlike_seviyesi)

    def analiz_et(self):
        self.stabilite -= 15
        print("Karanlık madde analiz edildi.")

    def acil_durum_sogutmasi(self):
        self.stabilite += 50
        print("Karanlık madde soğutuldu!")

File: KuantumCS/KaranlikMadde.cs
using System;

public class KaranlikMadde : KuantumNesnesi, IKritik
{
    public KaranlikMadde(string id, int stabilite, int tehlike)
        : base(id, stabilite, tehlike)
    {
    }

    public override void AnalizEt()
    {
        Stabilite -= 15;
        Console.WriteLine("Karanlık madde analiz edildi.");
    }

    public void AcilDurumSogutmasi()
    {
        Stabilite += 50;
        Console.WriteLine("Karanlık madde soğutuldu!");
    }
}
